pkg/api/datasources.go

```diff
@@ -483,3 +488,10 @@ func (hs *HTTPServer) CheckDatasourceHealth(c *models.ReqContext) response.Respo
 
 	return response.JSON(200, payload)
 }
+
+func (hs *HTTPServer) decryptSecureJsonData() func(map[string][]byte) map[string]string {
+	return func(m map[string][]byte) map[string]string {
+		decryptedJsonData, _ := hs.EncryptionService.DecryptJsonData(context.Background(), m, setting.SecretKey)
```

Comment:
What would it take to not ignore this error?

Reply:
Can we _at least_ log it?

Reply:
Done. Thanks for noticing this.